frontend/src/pages/Browse/Browse.test.tsx

```diff
@@ -1,30 +1,27 @@
 import { screen } from "@testing-library/react";
+import { vi } from "vitest";
 import { axe } from "vitest-axe";
 
 import { renderWithRouter } from "@/lib/test/renderWithRouter";
 
 import { Browse } from "./Browse";
 
+vi.mock("@/components/ProductColumn", () => ({
+  ProductColumn: () => (
+    <div data-testid="supermarket-container">Mocked ProductColumn</div>
+  ),
+}));
+
 describe("Given a user has gone to the browse page", () => {
```

Comment:
Small text change (not testing their navigation!) 

```suggestion
describe("Given a user is on the browse page", () => {
```